PeerioTechnologies/peerio-mobile
# feat: allowed delete in inline files and images [branch ch4386]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/4386/mobile-add-delete-file-to-inline-files-actions-like-in-desktop
#### Testing instructions  
1- Please test for inline files and images (two separate but similar looking action sheets are used)
2- Please note, inline file/image delete is only shown when the file exists in the user's fileStore. This is done because I encountered an instance where I tried deleting without the file existing in the user's fileStore and ran into an error. I then disabled the inline delete button when the file does not exist in fileStore, but haven't been able to reproduce the bug described above. Please try to reproduce a situation where an inline file is not in fileStore, and verify that the delete button is not shown in the action sheet. If you run into a situation where the delete button is shown but crashes or doesn't work as intended, it means the fix did not work and something else is causing the bug.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
